.github/ISSUE_TEMPLATE/accessibility.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Sweet, this issue templates will be really nice to have! 👍

Reply:
yeah> I copied EVERYTHING over pretty much so some things will need to be tweaked or reworded to encompass both types of components. I'll continue to work on that stuff but the Storybook is working for testing things out.